### ISSUE TITLE
About the learning rate

### ISSUE THREAD

**Issue by hanssssssss** [microsoft/Deep3DFaceReconstruction#137]
Hi ! I tried to overfit a single image and I found that in my training pieline the output of the model trained under the the learning rate 1e-5 performed much better than that trained under the learning rate 1e-4. Should I used the learning rate of 1e-5 when I train  on a bigger dataset(over 10k images),or 1e-4 is more suitable when training on a large amount of data? 
Thanks !
